.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build:
+    docker:
+      # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang:1.10-stretch
```

Comment:
FMI what does circle-ci images provides better than official golang images ?

Reply:
No idea. Just using what CircleCI proposes in their documentation here.